vta/src/runtime.cc

```diff
@@ -443,9 +443,18 @@ class UopQueue : public BaseQueue<VTAUop> {
       sram_begin_ = sram_end_;
     }
   }
+
```

Comment:
Remove spacing (too keep it consistent with rest of function definitions

Reply:
fixed.